tests/rbac/common/role/confirm_admin_test.py

```diff
@@ -13,152 +13,136 @@
+        reason = helper.proposal.reason()
+        message = rbac.role.admin.confirm.make(
+            proposal_id=proposal_id, user_id=user_id, role_id=role_id, reason=reason
         )
         self.assertIsInstance(
             message, protobuf.role_transaction_pb2.ConfirmAddRoleAdmin
         )
-        self.assertEqual(message.user_id, user.user_id)
-        self.assertEqual(message.role_id, role.role_id)
+        self.assertEqual(message.proposal_id, proposal_id)
+        self.assertEqual(message.user_id, user_id)
+        self.assertEqual(message.role_id, role_id)
         self.assertEqual(message.reason, reason)
-        return message, keypair
 
-    @pytest.mark.unit
+    @pytest.mark.library
     def test_make_addresses(self):
         """Test making the message addresses"""
-        self.assertTrue(callable(self.role.admin.confirm.make_addresses))
-        message, admin_key = self.test_make()
-
-        inputs, outputs = self.role.admin.confirm.make_addresses(
-            message=message, signer_keypair=admin_key
+        self.assertTrue(callable(rbac.role.admin.confirm.make_addresses))
+        user_id = helper.user.id()
+        role_id = helper.role.id()
+        proposal_id = helper.proposal.id()
+        proposal_address = rbac.role.admin.propose.address(role_id, user_id)
+        reason = helper.proposal.reason()
+        relationship_address = rbac.role.admin.address(role_id, user_id)
+        signer_keypair = helper.user.key()
+        signer_admin_address = rbac.role.admin.address(
+            role_id, signer_keypair.public_key
         )
-
-        relationship_address = addresser.role.admin.address(
-            message.role_id, message.user_id
+        message = rbac.role.admin.confirm.make(
+            proposal_id=proposal_id, user_id=user_id, role_id=role_id, reason=reason
         )
-        proposal_address = addresser.proposal.address(
-            object_id=message.role_id, target_id=message.user_id
+
+        inputs, outputs = rbac.role.admin.confirm.make_addresses(
+            message=message, signer_keypair=signer_keypair
         )
 
         self.assertIsInstance(inputs, list)
-        self.assertIsInstance(outputs, list)
+        self.assertIn(signer_admin_address, inputs)
         self.assertIn(proposal_address, inputs)
+        self.assertEqual(len(inputs), 2)
+
+        self.assertIsInstance(outputs, list)
         self.assertIn(relationship_address, outputs)
         self.assertIn(proposal_address, outputs)
         self.assertEqual(len(outputs), 2)
 
-    @pytest.mark.unit
+    @pytest.mark.library
     def test_make_payload(self):
         """Test making the message payload"""
-        self.assertTrue(callable(self.role.admin.confirm.make_payload))
-        message, admin_key = self.test_make()
-        payload = self.role.admin.confirm.make_payload(
-            message=message, signer_keypair=admin_key
+        self.assertTrue(callable(rbac.role.admin.confirm.make_payload))
```

Comment:
This line is not needed.  As the test below it is testing the 'callability' . of that method